readme.md

```diff
@@ -81,7 +81,8 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+pbj = Sandwich.new(type: "Peanut Butter and Jelly", id: 4 (assuming Geraldos id is 4))
+pbj.save()
 
```

Comment:
You virtually never want to hardcode IDs. Take a look at the [solution code](https://github.com/ga-dc/wdi7_quiz10/tree/solution#question-6).